### PR TITLE
localization, on top of fixing some translation errors. / turkish

### PR DESCRIPTION
**Pull request by hyesmo**

### CODE REVIEW

File: src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt [hunk 1]
```diff
@@ -442,11 +442,11 @@ sealed class Strings(
         installationSuccess = "✅ Yükleme başarıyla gerçekleşti",
         installed = "%0 Yüklendi",
         language = "Dil",
-        logging = "Günlük dosyasını yazmak",
-        modifyInstallation = "Kurulumu değiştir",
+        logging = "Log kaydı alınıyor",
```
Comment (hyesmo) [inline src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt:445]: did a mistake here, "Log kaydı al" is a better translation.

File: src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt [hunk 2]
```diff
@@ -411,29 +411,29 @@ sealed class Strings(
...
         downloadPreReleaseVersions = "Araçların yayınlanma öncesi sürümlerini indirin",
         error = "❌ Hata",
-        extraInventoryItems = "Oyun içi bir Steam envanterine fazladan öğeler ekleyin",
+        extraInventoryItems = "Oyun içi bir Steam envanterine ekstra eşyalar ekleyin",
         fetchingToolInfo = "%0 bilgisi getiriliyor",
         gameMode = "Oyun Modu",
-        inDevelopment = "Geliştirme aşamasında",
+        inDevelopment = "Geliştirilme aşamasında",
         information = "Bilgi",
         install = "Yükle",
```
Comment (hyesmo) [inline src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt:438]: "Kur" for install and "Kurulum" for installation is better.

File: src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt [hunk 3]
```diff
@@ -411,29 +411,29 @@ sealed class Strings(
...
+        checkForUpdates = "Güncelleştirmeleri kontrol et",
         clearCache = "Önbelleği temizle",
-        computing = "Bilgi İşlemi",
+        computing = "İşleniyor",
         configuration = "Yapılandırma",
-        confirmForceModifyInstallationMessage = "%0 işlemi şu anda sisteminizde çalışmakta olduğu bulundu. " +
+        confirmForceModifyInstallationMessage = "%0 işlemi şu anda sisteminizde çalışıyor. " +
                 "Yükleme durumunu değiştirmek için bu işlemi kapatmak gerekir. " +
-                "Zorla kapatarak yüklemeye devam etmek istiyor musunuz?",
+                "İşlemi zorla kapatarak yüklemeye devam etmek istiyor musunuz?",
```
Comment (hyesmo) [inline src/jvmMain/kotlin/acidicoala/koalageddon/core/values/Strings.kt:427]: İşlemi zorla kapatarak **_kuruluma_** devam etmek istiyor musunuz?

**Comment by CanerKaraca23** [comment]
Thanks for corrections.

**Comment by acidicoala** [comment]
Thank you!

**Pull request merged**